docs/setup/cookbook/infrastructure-and-requirements.md

```
@@ -109,6 +109,8 @@ Residents and operators need a small fleet of phones to exercise INJI Wallet on
 
 MOSIP supplies the platform and integrates the third-party components. The country team is responsible for the operating environment, gateways and identity-related accounts.
 
+![](../../.gitbook/assets/fpg-figure-2.png)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)